larray/core.py

```diff
@@ -7043,40 +7043,40 @@ def percentile_by(self, q, *args, **kwargs):
         Examples
         --------
         >>> arr = ndtest((4, 4), dtype=float)
-        >>> arr[:,:] = [[10, 7, 5, 9], \
-                        [5, 8, 3, 7], \
-                        [6, 2, 0, 9], \
-                        [9, 10, 5, 6]]
+        >>> arr[:,:] = [[10, 5, 7, 12], \
+                        [5, 8, 7, 9], \
+                        [5, 5, 3, 9], \
+                        [10, 8, 3, 12]]
         >>> arr
-        a\\b |   b0 |   b1 |  b2 |  b3
-         a0 | 10.0 |  7.0 | 5.0 | 9.0
-         a1 |  5.0 |  8.0 | 3.0 | 7.0
-         a2 |  6.0 |  2.0 | 0.0 | 9.0
-         a3 |  9.0 | 10.0 | 5.0 | 6.0
+        a\\b |   b0 |  b1 |  b2 |   b3
+         a0 | 10.0 | 5.0 | 7.0 | 12.0
+         a1 |  5.0 | 8.0 | 7.0 |  9.0
+         a2 |  5.0 | 5.0 | 3.0 |  9.0
+         a3 | 10.0 | 8.0 | 3.0 | 12.0
         >>> arr.std()
-        2.822205476218909
+        2.7810744326608736
         >>> # along axis 'a'
-        >>> round(arr.std('a'), 2)
-        b |   b0 |   b1 |   b2 |  b3
-          | 2.06 | 2.95 | 2.05 | 1.3
+        >>> arr.std('a')
+        b |  b0 |  b1 |  b2 |  b3
+          | 2.5 | 1.5 | 2.0 | 1.5
         >>> # along axis 'b'
-        >>> round(arr.std('b'), 2)
-        a |   a0 |   a1 |   a2 |   a3
-          | 1.92 | 1.92 | 3.49 | 2.06
+        >>> arr.std('b')
+        a |            a0 |            a1 |            a2 |            a3
+          | 2.69258240357 | 1.47901994577 | 2.17944947177 | 3.34477204006
         >>> # select rows a0 and a1 (ignore rows a2 and a3)
-        >>> round(arr.std('a0,a1'), 2)
+        >>> arr.std('a0,a1')
         b |  b0 |  b1 |  b2 |  b3
-          | 2.5 | 0.5 | 1.0 | 1.0
+          | 2.5 | 1.5 | 0.0 | 1.5
         >>> # split axis 'a' in two parts
-        >>> round(arr.std('a0,a1;a2,a3'), 2)
+        >>> arr.std('a0,a1;a2,a3')
           a\\b |  b0 |  b1 |  b2 |  b3
-        a0,a1 | 2.5 | 0.5 | 1.0 | 1.0
-        a2,a3 | 1.5 | 4.0 | 2.5 | 1.5
+        a0,a1 | 2.5 | 1.5 | 0.0 | 1.5
+        a2,a3 | 2.5 | 1.5 | 0.0 | 1.5
```

Comment:
It is a bit annoying that the two groups have the same result. I am unsure it's worth changing though...